pipeline/src/infer/InferenceConfig.jl

```diff
@@ -19,7 +19,7 @@ struct InferenceConfig{T, F, I, L, E}
     "Latent model type."
     latent_model::L
     "Case data"
-    case_data::Union{Vector{Integer}, Missing}
+    case_data::Union{Vector{Union{Integer, Missing}}, Missing}
```

Comment:
are these actually related to getting the pipeline working with the latest `EpiAware` version?

Reply:
Yes.